docs/general/administration/hardware-selection.md

```diff
@@ -50,6 +50,21 @@ If you are planning to use a dedicated graphics card (including upgrading an old
 
 Intel Drivers are much easier to install on Linux, with many distributions including them by default. If you do not need CUDA for other applications, it is highly recommended that you stick with Intel Graphics on Linux.
 
+### Servers without GPUs
+
+Not having a GPU is **NOT** recommended for Jellyfin. The following specs are provided for reference only.
+
+<details>
+- RAM: 16GB
+
+For software tone-mapping 4K 24fps Dolby Vision content to SDR:
+CPU: AMD Ryzen 7 5700X, Intel Core i5-12400F or better
+
+For 4K 60fps Dolby Vision content:
+CPU: AMD Ryzen 9 5950X, Intel Core i7-12700F or better
+
+</details>
```

Comment:
I'd say this is still inaccurate. 12700F has no way to guarantee 4K60 dolby vision and choosing CPU solely by dolby vision content performance is still very biased because as I said earlier it is probably the worst thing we can have. Dolby Vision videos has no fallback (like profile 5 videos) will have a significant impact on performance to a point that you will need 8 core zen3 for 4K24 playback while normal HDR10 videos or dolby vision videos with fallback (like profile 8.1) can be tone mapped with only 4 core Zen3 and still get 4K24 playback. These numbers are encoding with libx264 veryfast, using libx265 or libsvtav1 may reduce the framerate even further, so don't try to make a "recommended spec" because we cannot as it is way too scenario specific. 

We also need to be clear about the following points why CPU only is not recommended:

- Transcoding videos with CPU is performance demanding, and it is hard to gurantee all clients are compatible with all media sources
- Advanced codecs like HEVC and AV1 is even slower
- HDR to SDR tone mapping can make things worse, depending on the inputs you may end up with need 16 core zen3 level performance, or even no desktop CPUs currently can handle(if you want to use libx265 for dolby vision 4K60)

So if we have to do hardware selection guide on CPUs, we just say as much CPU cores as possible because the range is just too wide. The CPU only needs raspberry pi level performance if no video transcoding is needed, and a 4-6 core zen 3 can handle most use cases, including HDR10 4k24 tone mapping with libx264 encoder, but the CPU requirement goes up significantly when libx265/libsvtav1 is used and/or dolby vision reshaping is required.